examples/dao/assets/proposal-lsig.py

```diff
@@ -0,0 +1,57 @@
+import sys
+sys.path.insert(0,'..')
+
+from algobpy.parse import parse_params
+from pyteal import *
+
+def proposal_lsig(ARG_OWNER):
+    """
+    Represents Proposal Lsig (a contract account owned by the proposer)
+    """
+
+    # check no rekeying and ensure sender is owner
+    def basic_owner_checks(txn: Txn): return And(
+        txn.rekey_to() == Global.zero_address(),
+
+        # only owner can close ASA/ALGO
+        txn.sender() == Addr(ARG_OWNER)
+    )
+
+    def basic_checks(txn: Txn): return And(
+        txn.rekey_to() == Global.zero_address(),
+        txn.close_remainder_to() == Global.zero_address(),
+        txn.asset_close_to() == Global.zero_address()
+    )
+
+    # Opt-in transaction.
+    opt_in = And(
+        basic_checks(Txn),
+        Txn.type_enum() == TxnType.AssetTransfer,
+        Txn.asset_amount() == Int(0),
+        Txn.sender() == Addr(ARG_OWNER)
```

Comment:
why are we checking `Txn.sender` to be `ARG_OWNER` here ?

Reply:
updating to `asset_sender`